pkg/harvester/edit/loadbalancer.harvesterhci.io.ippool/PriorityRow.vue

```diff
@@ -53,8 +53,10 @@ export default {
 
     filteredNamespaces() {
       const namespaces = this.allNamespaces || [];
+      const selectedNamespaces = this.rows.map((row) => row?.namespace);
+      const filteredNamespaces = this.isStandaloneHarvester ? namespaces.filter((n) => !selectedNamespaces.includes(n.id) || n.id === this.row.namespace) : namespaces;
 
-      return namespaces.filter((namespace) => {
+      return filteredNamespaces.filter((namespace) => {
```

Comment:
Do we consider using disabled for already selected options instead of filtering? 
I feel this would be clearer for the user.
```
namespaceOptions() {
  const namespaces = this.filteredNamespaces;
  const selected = this.rows.map((r) => r?.namespace);

  if (!this.isStandaloneHarvester) {
    return [{
        label: this.t('generic.all'),
        value: '*'
      },
      ...namespaces.map((ns) => ({
        label: ns.metadata.name,
        value: ns.id,
        disabled: selected.includes(ns.id) && ns.id !== this.row.namespace,
      }))
    ];
  }

  return [{
      label: this.t('generic.all'),
      value: '*'
    },
    ...namespaces.map((ns) => ({
      label: ns.metadata.name,
      value: ns.id,
    }))
  ];
},
```